swagger/api/v1/definition_statuses.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  operationId: getPresentationDefinitionsStatus
+  summary: Gets the status of the submitted presentation(s) for the definition.
+  parameters:
+    - name: id
+      in: path
+      description: Presentation Definition ID
+      required: true
+      schema:
+        type: string
+  responses:
+    '200':
+      description: Status has been provided.
+      content:
+        application/json:
+          schema:
+            type: object
+            $ref: './api/models/status_wrapper.yaml#/presentation_status_wrapper'
+    '400':
```

Comment:
Given this is a REST API with an id in the path, I would also expect an explicit 404 response

Reply:
added